### PR TITLE
Bug 2090816: baremetal: wait longer for bootstrap to complete

### PR DESCRIPTION
**Pull request by honza**
This is a baremetal-specific alternative to #5979

### CODE REVIEW

File: cmd/openshift-install/create.go [hunk 1]
```diff
@@ -350,6 +350,16 @@ func waitForBootstrapComplete(ctx context.Context, config *rest.Config) *cluster
 	discovery := client.Discovery()
 
 	apiTimeout := 20 * time.Minute
+
+	// Wait longer for baremetal, due to length of time it takes to boot
+	if assetStore, err := assetstore.NewStore(rootOpts.dir); err == nil {
```
Comment (barbacbd) [inline cmd/openshift-install/create.go:355]: Same code is used in multiple places, should a function be used?

Comment (sadasu) [reply cmd/openshift-install/create.go:355]: Identical logic here https://github.com/openshift/installer/blob/master/cmd/openshift-install/create.go#L449-L455 while waiting for the cluster to be initialized.

**Pull request merged**